capstone/client/src/components/styles.css

```diff
@@ -343,12 +345,13 @@ h6 {
 }
 
 #back-button {
```

Comment:
Can we make the font of all the buttons Helvetica Neue?

Reply:
(or Google sans, as in the Google Material guide, if you can get it -- if not, no worries)